Source/WebKit/NetworkProcess/storage/NetworkStorageManager.h

```diff
@@ -247,7 +258,9 @@ class NetworkStorageManager final : public IPC::WorkQueueMessageReceiver {
     std::optional<double> m_originQuotaRatio;
     std::optional<double> m_totalQuotaRatio;
     std::optional<uint64_t> m_volumeCapacityOverride;
-    std::unique_ptr<NetworkQuotaManager> m_quotaManager;
+    std::optional<uint64_t> m_totalUsage;
+    std::optional<uint64_t> m_totalQuota;
```

Comment:
I guess we do not really need m_totalQuota to be an optional. It could be a uint64_t and 0 means uninitialized.